R/draws.R

```diff
@@ -9,9 +9,9 @@
 #' in the absence of intercurrent events (ICEs) handled using reference-based imputation methods.
 #' For this reason, any observed outcome data after ICEs, for which reference-based imputation methods are
 #' specified, are removed and considered as missing for the purpose of estimating the imputation model, and for
-#' this purpose only. The imputation model is a mixed effects model repeated measures (MMRM) model that is valid
+#' this purpose only. The imputation model is a mixed model for repeated measures (MMRM) that is valid
```

Comment:
Is "for" needed here? I thought it was just "Mixed model repeated measures" ?

Reply:
I think that this is also what more commonly you find in the literature...